Amazon.QLDB.Driver/exception/ExceptionMessages.cs

```diff
@@ -22,10 +22,7 @@ internal static class ExceptionMessages
         internal const string DriverClosed = "Operation is invalid as this QldbDriver has already been disposed.";
         internal const string FailedToSerializeParameter = "Error serializing statement parameters: ";
         internal const string ResultEnumeratorRetrieved = "The Enumerator of a Result can only be retrieved once. Please execute a new statement or buffer the results.";
-        internal const string SessionClosed = "Operation is invalid as this QldbSession has already been disposed.";
-        internal const string SessionPoolEmpty = "No sessions were returned to the pool and the timeout has been reached. Dispose some active sessions and retry.";
-        internal const string TransactionAlreadyOpened = "Transaction has already been opened.";
```

Comment:
Can we make a ticket to revisit the other drivers to ensure there are no dead messages left?

Reply:
Done.